typescript-playwright-sample/README.md

```diff
@@ -0,0 +1,73 @@
+# typescript-playwright-sample
+
+This sample demonstrates how you might set up a CI build for a simple, static html page to perform end to end accessibility tests in a browser, including how to suppress pre-existing or third-party failures. 
+
+This sample uses [Playwright](https://playwright.dev) for browser automation and uses the corresponding `@axe-core/playwright` library to integrate Axe and Playwright. But you don't have to use Playwright to use Axe! If you prefer a different browser automation tool, you can still follow the same concepts from this sample by using the integration library appropriate for your framework:
+
+* For **Selenium**, see our [typescript-selenium-webdriverjs-sample](../typescript-selenium-webdriverjs-sample/README.md)
```

Comment:
I think the links should reference `typescript-selenium-webdriver-sample` (no `js`)